src/main/java/internet/market/model/Order.java

```diff
@@ -7,12 +7,21 @@
     private Long id;
     private List<Product> products;
     private Long userId;
+    private double sum;
```

Comment:
let's remove this field - we will have troubles later when adding a DB connection
if you wish you can sum all prices inside your JSP

Reply:
done